openassessment/assessment/api/staff.py

```diff
@@ -56,6 +58,41 @@ def assessment_is_finished(submission_uuid, requirements):
     return True
 
 
+def on_start(submission_uuid):
+    """Create a new staff workflow for a student item and submission.
```

Comment:
Nit: my understanding from Ned is that our standard is to wrap the first line, even though that isn't the Google standard.